src/refresh/vkpt/shader/global_ubo.h

```diff
@@ -124,8 +121,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 	UBO_CVAR_DO(tm_hdr_peak_nits, 800.0) /* Exposure value 0 is mapped to this display brightness (post tonemapping) */ \
 	UBO_CVAR_DO(tm_hdr_saturation_scale, 100) /* HDR mode saturation adjustment, percentage [0..200], with 0% -> desaturated, 100% -> normal, 200% -> oversaturated */ \
 	UBO_CVAR_DO(ui_hdr_nits, 300) /* HDR mode UI (stretch pic) brightness in nits */ \
+	UBO_CVAR_DO(pt_particle_brightness, 100) /* particle brightness */ \
```

Comment:
Why change this?

Reply:
This was for an unrelated issue regarding RADV. The change was reverted in 1a888343e5c9077c004a7297855b9c1c2f9a2687.

Reply:
These kinds of changes should **usually** be included in the PR description. I like to have one commit per change and have a list of those commits in the PR description.